database/seeders/DatabaseSeeder.php

```diff
@@ -18,5 +18,6 @@ public function run(): void
         $this->call(CompanySeeder::class);
         $this->call(ProjectSeeder::class);
         $this->call(AdditionalTrainingSeeder::class);
+        $this->call(CollaborationSeeder::class);
```

Comment:
the develop branch has code on line 21, that's the reason for the merge conflict.

Reply:
when the conflict appears, you need to commit both entries in order to keep the old code and have the new one

Reply:
@JuanH941214 when a change request is resolve you should change the status to accepted